Sources/PIALibrary/Client+Preferences.swift

```diff
@@ -78,6 +79,7 @@ private protocol PreferencesStore: class {
     
 }
 
+@available(tvOS 17.0, *)
```

Comment:
@kp-laura-sempere Out of my own iOS ignorance. Why do we need to add this to all these methods? Isn't available by default?

Reply:
@kp-juan-docal , this makes the method only available on `tvOS 17+`
Some APIs from VPN connectivity are only available on `tvOS 17` but those objects are used in other classes too.
So in order to make the `PIALibrary` compile we need to add this annotations pretty much in every class.
We've had to do this because we made the `PIALibrary` available on tvOS 16 in order to make it compile with Xcode 14. These annotations would not be necessary if we had the `PIALibrary` minimum tvOS version set to 17, but we had to keep Xcode 14 compatibility for now.

Let me know if you have any other questions.